src/transformers/processing_utils.py

```diff
@@ -226,7 +460,19 @@ def from_pretrained(
             kwargs["token"] = token
 
         args = cls._get_arguments_from_pretrained(pretrained_model_name_or_path, **kwargs)
-        return cls(*args)
+
+        # Existing processors on the Hub don't have `processor_config.json`, but we need to keep `from_pretrained` work.
+        # This is not ideal (for models added in the future) as it might hide some bug/error silently.
+        # TODO: How to deal with this better and safer. Can we use timestamp as a condition to determine this?
+        try:
+            processor_dict, kwargs = cls.get_processor_dict(pretrained_model_name_or_path, **kwargs)
+        except EnvironmentError as e:
+            if "does not appear to have a file named processor_config.json." in str(e):
+                processor_dict, kwargs = {}, {}
+            else:
+                raise
+
+        return cls.from_args_and_dict(args, processor_dict, **kwargs)
```

Comment:
I would like to hear from the core maintainers to see if there is a better approach for this situation 🙏

Reply:
Tbh, I like this handling. It's backwards compatible and only sets a fallback behaviour if a specific error message is raised.